R/shared-methods.R

```diff
@@ -68,7 +68,7 @@ check_country_available <- function(country = character(), level = 1,
 #' @param totals Logical, defaults to FALSE. If TRUE, returns totalled
 #'  data per region up to today's date. If FALSE, returns the full dataset
 #'  stratified by date and region.
-#' @param localise Logical, defaults to TRUE. Should region names be localised.
+#' @param localise Logical, defaults to TRUE. Should region names be localized.
 #' @param verbose Logical, defaults to TRUE. Should verbose processing
 #' messages and warnings be returned.
 #' @param steps Logical, defaults to FALSE. Should all processing and cleaning
```

Comment:
I am not sure if this is mentioned but we should really document that data gets stored in a list which can be accessed using `$data`....